setup.py

```diff
@@ -24,10 +24,12 @@
 except:
     requirements.append("hashlib")
 
+import bagit
```

Comment:
Hmm, I was following [the section](https://www.python.org/dev/peps/pep-0008/#imports) "Imports should be grouped in the following order: 1. standard library imports 2. related third party imports 3. local application/library specific imports" and didn't see anything about `import x` coming before `from x import y`. Happy to move it all the way to the top if you want.